sphinx.go

```diff
@@ -530,11 +530,14 @@ func (r *Router) Stop() {
 // In the case of a successful packet processing, and ProcessedPacket struct is
 // returned which houses the newly parsed packet, along with instructions on
 // what to do next.
-func (r *Router) ProcessOnionPacket(onionPkt *OnionPacket,
-	assocData []byte, incomingCltv uint32) (*ProcessedPacket, error) {
+func (r *Router) ProcessOnionPacket(onionPkt *OnionPacket, assocData []byte,
```

Comment:
I like how we absolve the caller from having to compute any blinded ID key and keep that computation confined to the internals of this package. This keeps the top level API nice and clean: 
```
ProcessOnionPacket(pkt, ..., nil)
ProcessOnionPacket(pkt, ..., ephemeralBlindingPoint)
```

From a caller's perspective it calls the same function and includes the optional blinding point if it finds one in *`UpdateAddHTLC`*! This keeps the `lnd` diff smaller!